docs/content/privacy.md

```diff
@@ -4,24 +4,24 @@ kind: operations
 weight: 50
 ---
 
-This document provides details about OPA's anonymous version reporting feature.
+This document provides details about OPA's anonymous information reporting feature.
 
 ## Overview
 
-OPA periodically reports its version to a publicly hosted, external service. The version reports only contain the OPA
-version number (e.g., v0.12.3) and a randomly generated UUID. This feature is only applicable to the `opa run` and
-`opa version` commands.
+OPA periodically reports its version, memory usage to a publicly hosted, external service. The reports contain the OPA
+version number (e.g., v0.12.3), heap usage in bytes and a randomly generated UUID. This feature is only applicable to
+the `opa run` and `opa version` commands.
 
 In case of the `opa run` command, this feature is **ON by-default** and can be easily disabled by specifying
 the `--skip-version-check` flag. When OPA is started in either the server or repl mode, OPA calls the external service
```

Comment:
The flag name is misleading now. Perhaps we can add `--disable-telemetry` as a new flag, and check the old flag for compat?

Reply:
Added the new `--disable-telemetry` flag.